### PR TITLE
Cut V0.25.0 rc.0

### PR DESCRIPTION
**Pull request by simonpasquier**
@gotjosh I know you have https://github.com/prometheus/alertmanager/pull/3132 pending. Do you think that you have time to address my comments or can it wait for the next minor Alertmanager release?

### CODE REVIEW

**Comment by gotjosh** [comment]
I'll address your comments shortly!

**Review by roidelapluie — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+## 0.25.0-rc.0 / 2022-12-16
+
+* [CHANGE] Change the default `parse_mode` value from `MarkdownV2` to `HTML` for Telegram. #2981
+* [CHANGE] Make `api_url` field optional for Telegram. #2981
+* [CHANGE] Use CanonicalMIMEHeaderKey instead of TitleCasing for email headers. #3080
+* [CHANGE] Reduce the number of notification logs broadcasted between peers by expiring them after (2 * repeat interval). #2982
+* [CHANGE/BUGFIX] Fail configuration loading if `api_key` and `api_key_file` are defined at the same time. #2910
```
Comment (roidelapluie) [inline CHANGELOG.md:7]: We generally don't mix, so this should be a BUGFIX only I think

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+* [CHANGE/BUGFIX] Fix the `alertmanager_alerts` metric to avoid counting resolved alerts as active. Also added a new `alertmanager_marked_alerts` metric that retain the old behavior. #2943
```
Comment (roidelapluie) [inline CHANGELOG.md:8]: This is a CHANGE only

File: CHANGELOG.md (hunk 1 above)
Comment (gotjosh) [reply CHANGELOG.md:7]: Ditto, this looks like a bugfix.

Comment (simonpasquier) [reply CHANGELOG.md:7]: I thought that Alertmanager v0.24.0 would have accepted `api_key` and `api_key_file` but in fact setting `api_key_file` failed the configuration loading...

**Review by gotjosh — APPROVED** [review]
LGTM, but please see my comments.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+* [FEATURE] Add `proxy_url` support for OAuth2 in HTTP client configuration. #3010 (0.33)
```
[the remark this answers is not in the document]
Comment (simonpasquier) [reply CHANGELOG.md:7]: my bad, some leftover :)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,37 @@
...
+* [ENHANCEMENT] Add `--web.systemd-socket` flag to systemd socket activation listeners instead of port listeners (Linux only). #3140
+* [ENHANCEMENT] Add `enable_http2` support in HTTP client configuration. #3010 (0.33)
+* [ENHANCEMENT] Add `min_version` support to select the minimum TLS version in HTTP client configuration. #3010 (0.34)
+* [ENHANCEMENT] Add `max_version` support to select the maximum TLS version in HTTP client configuration. #3168
+* [ENHANCEMENT] Emit warning logs when truncating messages in notifications. #3145
+* [ENHANCEMENT] Add `--data.maintenance-interval` flag to define the interval between the garbage collection and snapshotting to disk of the silences and the notification logs. #2849
+* [ENHANCEMENT] Support HEAD method for the `/-/healty` and `/-/ready` endpoints. #3039
+* [ENHANCEMENT] Truncate messages with the `…` ellipsis character instead of the 3-dots string `...`. #3072
+* [ENHANCEMENT] Add support for reading global and local SMTP passwords from files. #3038
+* [ENHANCEMENT] Add location support to time intervals. #2782
```
[the remark this answers is not in the document]
Comment (simonpasquier) [reply CHANGELOG.md:20]: I chose location based on Beorn's [comment](https://github.com/prometheus/alertmanager/pull/2782#issuecomment-1024921738) in the PR.

[the remark this answers is not in the document]
Comment (gotjosh) [reply CHANGELOG.md]: wow - nice read. TIL

**Review by gotjosh — APPROVED** [review]

**Pull request merged**